pkg/sloop/store/typed/eventcounttablegen.go

```diff
@@ -132,6 +132,22 @@ func (t *ResourceEventCountsTable) GetMinMaxPartitions(txn badgerwrap.Txn) (bool
 	return true, minKey.PartitionId, maxKey.PartitionId
 }
 
+func (t *ResourceEventCountsTable) GetMinPartitions(txn badgerwrap.Txn) (bool, string) {
```

Comment:
minor: It is only returning a singular partition, so it should not end with `s`

Reply:
Can we add a method GetMaxPartition, then tweak GetMinAndMaxPartitions to just call both of these, so internally we dont have code duplication?

Reply:
sure